doc/source/whatsnew/v0.17.0.txt

```diff
@@ -110,6 +110,7 @@ Other enhancements
 
 - ``pd.merge`` will now allow duplicate column names if they are not merged upon (:issue:`10639`).
 
+- ``msgpack`` submodule updated to 0.4.6 (:issue:`10581`)
 .. _whatsnew_0170.api:
```

Comment:
maybe show an example of where this fails / different. I don't think unpacking via regular msgpack is tested at all. I mean it would work, but give sort of the internal repr.